content/asset-tracking/using-the-sdks.textile

```diff
@@ -0,0 +1,232 @@
+---
+title: Using the SDKs
+section: asset-tracking
+index: 20
+meta_description: "Ably Asset Tracking SDKs provide an easy way to track multiple assets with realtime location updates powered by the Ably realtime network and Mapbox Navigation SDK with location enhancement."
+meta_keywords: "Ably, Asset Tracking SDK, Mapbox, Mapbox Navigation SDK, Ably realtime"
+languages:
+  - javascript
+  - android
+  - objc
+  - java
+  - swift
+jump_to:
+  Help with:
+    - Supported platforms#supported-platforms
+    - Repositories#repositories
+    - Prerequisites#prerequisites
+    - Installing the SDK#installing-sdk
+    - Using the Publisher SDK#publisher-sdk
+    - Using the Subscriber SDK#subscriber-sdk
+    - See also#see-also
+---
+
+This topic explains how to get started using the Ably Asset Tracking SDKs.
+
+h2(#supported-platforms). Supported platforms
+
+There are two Asset Tracking SDKs, one for publishing and one for subscribing. The following platforms are supported:
+
+* Android (Java and Kotlin) - publisher and subscriber SDK
```

Comment:
The terminology that we use at the moment is "publishing" and "subscribing" SDKs, not "publisher" and "subscriber" (@QuintinWillison correct me if I am wrong)

Reply:
Thanks for clarifying. Also changed in overview.